pipelines/managed/rh-advisories/rh-advisories.yaml

```diff
@@ -486,6 +486,8 @@ spec:
           value: "$(params.taskGitUrl)"
         - name: taskGitRevision
           value: "$(params.taskGitRevision)"
+        - name: retries
```

Comment:
I agree that it's kind of pointless to have the param, but never use it. But would it make more sense to simply change the default in the task to 3? Because this task is used by other pipelines as well. And it seems weird to only enable retries in this one pipeline.

And also, note that this and other pipelines have retries enabled on the pipeline level, meaning that this whole task will be retried up to 5 times. Did that not work?

Reply:
Good point about multiple pipelines using it and just changing the task default

Reply:
Sure, sounds good.